pkg/dev_compiler/tool/global_compile.dart

```diff
@@ -60,7 +60,7 @@ void main(List<String> args) {
     '--modules=legacy', // TODO(vsm): Change this to use common format.
     '--single-out-file',
     '--inline-source-map',
-    '-p',
+    '--packages',
```

Comment:
Was this an accidental change?  It doesn't seem like it belongs in this pull request